tools/marine_omics/sanntis.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="sanntis_marine" name="Sanntis biosynthetic gene clusters" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
     <description>in genomic and metagenomic data</description>
     <macros>
-        <token name="@TOOL_VERSION@">0.9.3.5</token>
+        <token name="@TOOL_VERSION@">0.9.3.6</token>
```

Comment:
Is that one already available? https://anaconda.org/search?q=sanntis

What I meant is the suffix version.

Reply:
oups